module-2/backend/week_one/Ash.md

```diff
@@ -0,0 +1,104 @@
+## Week One - Module 2 Recap
+
+List the five common HTTP verbs and what the purpose is of each verb.
+  Get= Read part of CRUD
+  Post= Create part of CRUD
+  Put= Update part of CRUD
+  Delete= Delete part of CRUD
+
+What is Sinatra?
+  A framework which allows you to deploy an app/site, simple but effective, not as robust as Rails but will be recommended in certain situations.
+What is MVC?
+  Model / View / Controllers

```

Comment:
This is what it stands for. What is it? A pattern that we use to organize code in web applications.